### ISSUE TITLE
'nodeId' parameter on dht.addNode is required

### ISSUE THREAD

**Issue by moshest** [feross/bittorrent-dht#47]
But on the [documentation](https://github.com/feross/bittorrent-dht#dhtaddnodeaddr-nodeid) it's say it optional.

Example:
```
dht = new DHT({bootstrap: false})
dht.addNode('127.0.0.1:20000') // throws an error!
```

```
TypeError: First argument needs to be a number, array or string.
    at new Buffer (buffer.js:188:15)
    at idToBuffer (node_modules\bittorrent-dht\client.js:1196:12)
    at DHT.addNode (node_modules\bittorrent-dht\client.js:278:12)
    ...
```

**Comment by feross** [comment]
Thanks for the bug report. I was aware of this issue, but there wasn't an issue here to track it.

There's a TODO in the webtorrent code here: https://github.com/feross/webtorrent/blob/3f75dbd2f735f45a19c3cd679b6f957baf0071c6/lib/torrent.js#L458

**Comment by No9** [comment]
Hey @feross I have just run into this one too. 
The error I am seeing seems to be a little further on and different than the one mentioned by @moshest so I wondered if this was meant to be fixed? 

```
scuttlebot\node_modules\bittorrent-dht\client.js:301
if (nodeId.length !== 20) throw new Error('invalid node id length')
```

Thanks

**Comment by feross** [comment]
Thanks for reporting this issue, @moshest and @No9. This is fixed and released as 3.1.0.
